src/python/pants/backend/jvm/targets/jvm_target.py

```diff
@@ -126,6 +133,16 @@ def strict_deps(self):
     return self.payload.strict_deps
 
   @property
+  def exports(self):
+    """A list of exported libraries, which will be accessible to dependents.
+
+    :return: See constructor.
+    :rtype: list
+    """
+    exports = [Address.parse(spec, relative_to=self.address.spec_path) for spec in self.payload.exports]
+    return [self._build_graph.get_target(addr) for addr in exports]
```

Comment:
Yea, and now I'm less sure about it. You're right that folks would begin to just use it by default.